docs/workflow.md

```diff
@@ -49,11 +49,11 @@ For each Issue of project you are working on, you should have a separate branch.
 
 1. Navigate to the base folder of your Software repository: `cd path/to/the/repository/Software`
 2. Make git aware of any new changes to `upstream` by running `git fetch upstream`
-3. Create a new branch from `upstream/master` by running `git checkout -b your-branch-name upstream/master`
+3. Create a new branch from `upstream/master` by running `git checkout upstream/master` then `git checkout -b your-branch-name`
```

Comment:
Is there any difference between these two commands?

Reply:
Yeah `git checkout -b your-branch-name upstream/master` causes `your-branch-name` to track `upstream` so it creates a branch on https://github.com/UBC-Thunderbots/Software instead of on people's fork. 
I changed the workflow to how I do it. Wondering what your workflow looks like